bindings/java/src/main/native/Makefile.am

```diff
@@ -43,7 +43,7 @@ libjucx_la_SOURCES = context.cc \
                      ucs_constants.cc \
                      worker.cc
 
-libjucx_la_CXXFLAGS = -fPIC -DPIC -Werror -std=c++11
+libjucx_la_CXXFLAGS = -fPIC -DPIC -Werror
```

Comment:
BTW, shall we add BASE_CXXFLAGS here, which also have -O3 -Wall -g (could be another PR)